### ISSUE TITLE
variables messed

### ISSUE THREAD

**Issue by ashfor03** [rajkosto/hive#3]
```
database.dll - 0.9.8.4
hiveEXT.dll - 0.9.6.6

rpt
17:57:03 "HIVE: WRITE: "CHILD:306:1081:[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]:0.0511811:""
17:57:03 "WRITE: ->["PASS"][CHILD:306:1081:[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]:0.0511811:]"
17:57:05 "HIVE: WRITE: "CHILD:305:1081:[94,[8994.75,5386.68,0.00531006]]:0.356997:""
17:57:05 "WRITE: ->["PASS"][CHILD:305:1081:[94,[8994.75,5386.68,0.00531006]]:0.356997:]"

hiveEXT.log
2012-11-16 17:57:03 Database.MySqlPreparedStatement: [Error] SQL: cannot execute 'UPDATE `Object_DATA` SET `Hitpoints` = ? , `Damage` = ? WHERE `ObjectID` = ?'  VALUES("[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]", 0.051181100308895111, 1081), ERROR Data too long for column 'Worldspace' at row 1
2012-11-16 17:57:03 Database.MySqlPreparedStatement: [Error] SQL: cannot execute 'UPDATE `Object_DATA` SET `Hitpoints` = ? , `Damage` = ? WHERE `ObjectID` = ?'  VALUES("[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]", 0.051181100308895111, 1081), ERROR Data too long for column 'Worldspace' at row 1
2012-11-16 17:57:05 Database.MySqlPreparedStatement: [Error] SQL: cannot execute 'UPDATE `Object_DATA` SET `Hitpoints` = ? , `Damage` = ? WHERE `ObjectID` = ?'  VALUES("[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]", 0.051181100308895111, 1081), ERROR Data too long for column 'Worldspace' at row 1
2012-11-16 17:57:05 Database.MySqlPreparedStatement: [Error] SQL: cannot execute 'UPDATE `Object_DATA` SET `Hitpoints` = ? , `Damage` = ? WHERE `ObjectID` = ?'  VALUES("[["sklo predni P",1],["sklo predni L",1],["glass2",1],["glass3",1]]", 0.051181100308895111, 1081), ERROR Data too long for column 'Worldspace' at row 1

mysql-query.log
1829 Execute UPDATE `Object_DATA` SET `Worldspace` = '[[\"sklo predni P\",1],[\"sklo predni L\",1],[\"glass3\",1]]' , `Fuel` = 0.05118110030889511 WHERE `ObjectID` = 1081
1829 Execute UPDATE `Object_DATA` SET `Worldspace` = '[[\"sklo predni P\",1],[\"sklo predni L\",1],[\"glass3\",1]]' , `Fuel` = 0.05118110030889511 WHERE `ObjectID` = 1081

```

**Comment by rajkosto** [comment]
Should be fixed in 00772db78af6e60dd1e14abce9b400b67e88c0d7 please test.

**Comment by rajkosto** [comment]
Confirmed fixed.
